.github/workflows/release-please.yml

```
@@ -27,8 +28,6 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: lts/*
+          node-version: 24.x
           registry-url: 'https://registry.npmjs.org'
       - run: npm publish --access public
```

nit: we might as well
```suggestion
      - run: npm publish --access public --provenance
```